cpp/ql/src/Security/CWE/CWE-022/TaintedPath.qhelp

```diff
@@ -3,36 +3,57 @@
   "qhelp.dtd">
 <qhelp>
 <overview>
-<p>Accessing paths controlled by users can allow an attacker to access unexpected resources. This
+<p>Accessing paths controlled by users can allow an attacker to access unexpected resources. This 
```

Comment:
I know this existed before. Just wondering if we could replace one of `Accessing` and `to access` with a different verb, to avoid repetition. I can't find anything better than `to gain entry` (to replace `to access`), so feel free to ignore if you think that there aren't any good alternatives 😅

Reply:
I don't think `to gain entry` is better in that sentence, so I'm keeping it.